python/dask_cudf/dask_cudf/expr/_collection.py

```diff
@@ -0,0 +1,108 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+
+from functools import cached_property
+
+from dask_expr import (
+    DataFrame as DXDataFrame,
+    FrameBase,
+    Index as DXIndex,
+    Series as DXSeries,
+    get_collection_type,
+)
+from dask_expr._collection import new_collection
+from dask_expr._util import _raise_if_object_series
+
+from dask import config
+from dask.dataframe.core import is_dataframe_like
+
+import cudf
+
+##
+## Custom collection classes
+##
+
+
+class CudfMixin:
+    def var(
```

Comment:
Dask-expr's `var` implementation doesn't work for cudf. May be able to upstream a "fix" for later dask-expr versions.

Reply:
OK, so this is because column-indexing of a DataFrame doesn't handle device-side objects, which seems like a missing cudf feature

Reply:
Linked an issue for this.